frontend/src/app/components/homescreen/blocks/new-features.component.ts

```diff
@@ -30,6 +30,7 @@ import {Component} from '@angular/core';
 import {DynamicBootstrapper} from "core-app/globals/dynamic-bootstrapper";
 import {I18nService} from "core-app/modules/common/i18n/i18n.service";
 import {DomSanitizer} from "@angular/platform-browser";
+import {BcfRestApi} from "core-app/modules/bim/bcf/bcf-constants.const";
 
 export const homescreenNewFeaturesBlockSelector = 'homescreen-new-features-block';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/1d1aa9e177ef71410aa0b85062889ff3a6c68b8f/frontend/src/app/components/homescreen/blocks/new-features.component.ts#L36"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>